up42/job.py

```diff
@@ -346,3 +346,17 @@ def get_jobtasks_results_json(self) -> dict:
 
             jobtasks_results_json[jobtask_id] = response_json
         return jobtasks_results_json
+
+    def get_job_credits(self) -> dict:
```

Comment:
Should we lose the "job" in the function name since the object is already job? `job.get_credit_costs()` or `job.get_credits()` etc.?

For "sound" reference, we probably will add some endpoints like `up42.get_credit_history()` as well.